src/routing/approved.rs

```diff
@@ -895,6 +901,14 @@ impl Approved {
 
     fn handle_join_request(&self, peer: Peer, join_request: JoinRequest) -> Result<Vec<Command>> {
         debug!("Received {:?} from {}", join_request, peer);
+        let pub_id = *peer.name();
+        if !self.join_flag {
+            debug!(
+                "Ignoring JoinRequest from {} - new node not acceptable.",
+                pub_id,
+            );
+            return Ok(vec![]);
+        }
```

Comment:
Now we are ignoring even relocated nodes, which probably isn't what we want.

Reply:
right, will change it so that relocated node will always be accepted.